modules/apps/commerce/commerce-product-service/src/main/java/com/liferay/commerce/product/service/impl/CPDefinitionLocalServiceImpl.java

```diff
@@ -2981,6 +2990,24 @@ private Map<String, String> _getUniqueUrlTitles(
 		return newURLTitleMap;
 	}
 
+	private void _isValidSubscriptionInfo(
+			int subscriptionLength, int deliverySubscriptionLength,
+			long maxSubscriptionCycles, long deliveryMaxSubscriptionCycles)
+		throws CPSubscriptionDateException {
+
+		if ((subscriptionLength < 1) || (deliverySubscriptionLength < 1)) {
+			throw new CPSubscriptionDateException(
+				"Enter subscription length greater than or equal to 1");
```

Comment:
@victorcdp , alter the messages as well to the requested format:

`Subscription length must be greater than or equal to 1`